frontend/src/metabase/lib/urls.js

```diff
@@ -49,6 +49,10 @@ export function question(card, hash = "", query = "") {
   return `${path}${query}${hash}`;
 }
 
+export function serializedQuestion(card) {
+  return question(null, card);
+}
+
```

Comment:
I just realized this file uses both major ways of declaring functions.

May be nice if FE team picks one way.

Fwiw, my personal preference would be fat arrow, with an implicit return in this case.
Small.

```suggestion
export const serializedQuestion = card => 
  question(null, card);
```

Reply:
We may want to introduce a typical naming convention for "static" consts and functions.

If function, we always start the name with a verb.

So it may typically look like this (imaginary situation):

```js
  const serializedQuestion = getSerializedQuestion(args)
  
  const getSerializedQuestion = args => { … }
```

when used consistently, it reveals intent that much faster.

Reply:
Maintaining consistency is good, but let's spawn the discussion to a separate thread/issue.
Bug fixing needs to focus on the smallest amount of code change that solves the problem.

Reply:
> Maintaining consistency is good, but let's spawn the discussion to a separate thread/issue.
> Bug fixing needs to focus on the smallest amount of code change that solves the problem.

Sure. Comment was meant for the future. 
So far I've been leaving approvals (when applicable of course) with occasional minor comments like this, definitely non-blocking.